core/urls.py

```diff
@@ -57,6 +57,7 @@
             extra_context={},
         ),
         name='user_accounts/password_reset/success'),
+<<<<<<< HEAD
```

Comment:
The account and password-change urls have been moved to `user_interaction/account_pages/config.py`. The account-change url can be moved there as well.